docs/quick-start.md

```diff
@@ -8,24 +8,85 @@ import TabItem from '@theme/TabItem';
 
 # Quick Start
 
-After [installing](./installation.md), use the following command to run it:
+Once you've [installed Yazi](./install.md), start the program with:
 
 ```bash
 yazi
 ```
 
 Press `q` to quit and `~` to open the help menu.
 
-## Themes
+## Shell function
+
+We suggest using this ya shell function that provides the ability to change the current working directory when exiting Yazi.
+
+<Tabs>
+  <TabItem value="bash-zsh" label="Bash / Zsh" default>
+
+```bash
+function ya() {
+	tmp="$(mktemp -t "yazi-cwd.XXXXX")"
+	yazi "$@" --cwd-file="$tmp"
+	if cwd="$(cat -- "$tmp")" && [ -n "$cwd" ] && [ "$cwd" != "$PWD" ]; then
+		cd -- "$cwd"
+	fi
+	rm -f -- "$tmp"
+}
+```
+
+  </TabItem>
+  <TabItem value="fish" label="Fish">
+
+```shell
+function ya
+	set tmp (mktemp -t "yazi-cwd.XXXXX")
+	yazi $argv --cwd-file="$tmp"
+	if set cwd (cat -- "$tmp"); and [ -n "$cwd" ]; and [ "$cwd" != "$PWD" ]
+		cd -- "$cwd"
+	end
+	rm -f -- "$tmp"
+end
+```
+
+  </TabItem>
+  <TabItem value="nushell" label="Nushell">
+
+```shell
+def --env ya [args?] {
+	let tmp = $"($env.TEMP)(char path_sep)yazi-cwd." + (random chars -l 5)
+	yazi $args --cwd-file $tmp
+	let cwd = (open $tmp)
+	if $cwd != "" and $cwd != $env.PWD {
+		cd $cwd
+	}
+	rm -f $tmp
+}
+```
 
-We have created a repo to collect themes from the community. Pick a theme you like!
+  </TabItem>
+  <TabItem value="powershell" label="PowerShell">
+
+```powershell
+function ya {
+    $tmp = [System.IO.Path]::GetTempFileName()
+    yazi $args --cwd-file="$tmp"
+    $cwd = Get-Content -Path $tmp
+    if (-not [String]::IsNullOrEmpty($cwd) -and $cwd -ne $PWD.Path) {
+        Set-Location -Path $cwd
+    }
+    Remove-Item -Path $tmp
+}
+```
+
+  </TabItem>
+</Tabs>
 
-https://github.com/yazi-rs/themes
+To use it, copy the function into the configuration file of your respective shell.
 
 ## Keybindings
 
 :::tip
-To see all key bindings, check the [yazi-config/preset/keymap.toml](https://github.com/sxyazi/yazi/blob/main/yazi-config/preset/keymap.toml) file.
+For all key bindings, see our [Keymap Reference](./configuration/keymap.md).
```

Comment:
The new link is just a list of commands rather than a list of keybindings

Reply:
Okay well that's why I think we need a reference section for that kind of thing. The current way we are explaining the configuration files doesn't work / isn't clear.

Reply:
How about:

> For all key bindings, see the [default `keymap.toml` file](https://github.com/sxyazi/yazi/blob/main/yazi-config/preset/keymap.toml) file.

Reply:
Sounds good to me, but not have the last "file"

Reply:
Oop syeah ahah.